### DISCUSSION TITLE
Welcome to arviz Discussions!

### DISCUSSION THREAD

**Discussion by aloctavodia** [arviz-devs/arviz#1463] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Note** If your question is also related to Stan or PyMC usage, consider asking on https://discourse.mc-stan.org/ or https://discourse.pymc.io/ respectively using the `arviz` tag.

**Comment by aloctavodia** [comment]
Testing, 1,2,3 can you hear me?

**Reply by abdz-128** [reply]
Hi guys,
New in the github community. I wanted to contribute to but I don't know where to get started. I would be pleased if someone guides me.
Cheers!!

**Reply by OriolAbril** [reply]
You should take a look at https://python.arviz.org/en/stable/contributing/index.html. We recently had a webinar on contributing to ArviZ too (also available in that page)
